packages/toolpad-studio/src/toolpad/AppOptions/index.tsx

```diff
@@ -28,7 +35,7 @@ function AppOptions({ dom }: AppOptionsProps) {
 
   return (
     <React.Fragment>
-      <IconButton {...buttonProps} aria-label="Application menu">
+      <IconButton {...(buttonProps as IconButtonProps)} aria-label="Application menu">
```

Comment:
We should probably fix the types of `useMenu` instead so that they work for both `Button` and `IconButton`.

Reply:
If I understand correctly, the issue has arisen because of the mismatch in the type of the `loading` prop between `[IconButton](https://github.com/mui/material-ui/blame/master/packages/mui-material/src/IconButton/IconButton.d.ts#L54)` and `[Button](https://github.com/mui/material-ui/blob/master/packages/mui-material/src/Button/Button.d.ts#L68)` and everything should work as expected when this issue is resolved without us needing to do anything

Reply:
Looks like it. Is there a core issue? We can hold off on upgrading until the issue is resolved

Reply:
Can't seem to find an issue for this. @siriwatknp is the difference in the types of `IconButton` and `Button` wrt `loading` intentional? (See my comment above in the thread for the links)